website/blog/2022-12-14-databend-weekly-72.md

```diff
@@ -0,0 +1,187 @@
+
+**Format**
+
+- support ASCII control code hex as format field delimiter ([#9160](https://github.com/datafuselabs/databend/pull/9160))
+
+**Planner**
+
+- prewhere_column empty and predicate is not const will return empty ([#9116](https://github.com/datafuselabs/databend/pull/9116))
+- don't push down topk to Merge when it's child is Aggregate ([#9183](https://github.com/datafuselabs/databend/pull/9183))
+- fix nullable column validity not equal ([#9220](https://github.com/datafuselabs/databend/pull/9220))
+
+**Query**
+
+- address unit test hang on test_insert ([#9242](https://github.com/datafuselabs/databend/pull/9242))
+
+**Storage**
+
+- too many io requests for read blocks during compact ([#9128](https://github.com/datafuselabs/databend/pull/9128))
+- collect orphan snapshots ([#9108](https://github.com/datafuselabs/databend/pull/9108))
+
+## What's On In Databend
+
+Stay connected with the latest news about Databend.
+
+#### Unifying Format Options and Removing `format_*` Settings
+
+> ***Breaking Change*** We plan to unify the formatting options and remove `format_*` settings in the future. 
+> This means that we will **disable** the old format.
+>  Work is in progress and support for `FILE_FORMAT` has now been completed in streming load.
+
+##### Propose
+
+In the new implementation, **only** use the snowflake style in any place (insert copy and "select from stage" later):
+
+- `FILE_FORMAT = (type = 'CSV' some_option = '|')`
+- `FILE_FORMAT = (format_name = 'MyCustomCSV')`
+
+`FORMAT CSV` is retained for the ClickHouse handler.
+
+MyCustomCSV is created using SQL like `CREATE FILE FORMAT`
+
+**Learn More**
+
+- [Issue | unify format options and remove format_ settings](https://github.com/datafuselabs/databend/issues/8995)
+- [PR | support file_format syntax in streaming load insert sql](https://github.com/datafuselabs/databend/pull/9063)
+
+#### Open Sharing
+
+Open Sharing is a cheap and secure data sharing protocol for databend query on multi-cloud environments.
+
+* **Cheap**: Open Sharing allow data sharing via simple RESTful API sharing protocol, which is cheap and easy to understand.
+* **Secure**: Open Sharing protocol would verify allow incoming requesters identity and access permission and provide audit log.
+* **Multi-cloud**: Open Sharing is designed to work with different cloud platforms, including AWS, Azure, GCP, etc.
+* **Open source**: Open Sharing is an open source project
+
+**Learn More**
+
+- [Docs | Open Sharing](https://github.com/datafuselabs/databend/blob/main/src/query/sharing-endpoint/README.md)
+- [PR | add prototype open sharing and add sharing stateful tests](https://github.com/datafuselabs/databend/pull/9177)
+
+## What's Up Next
+
+We're always open to cutting-edge technologies and innovative ideas. You're more than welcome to join the community and bring them to Databend.
+
+#### Refactoring Stage-Related Tests
+
+We should use streaming upload to put files directly in the stage instead of in s3.
```

Comment:
We're about to run stage-related tests again using the Streaming Load API to move files to a stage instead of an AWS command like this: